src/main/java/github/weichware10/util/config/CodeChartsConfiguration.java

```diff
@@ -60,17 +150,33 @@ public boolean equals(Object obj) {
         }
         CodeChartsConfiguration that = (CodeChartsConfiguration) (obj);
         return stringId.equals(that.stringId)
+                && Arrays.equals(timings, that.timings)
                 && Arrays.equals(initialSize, that.initialSize)
-                && Arrays.equals(timings, that.timings);
+                && (showGrid == that.showGrid)
+                && (relativeSize == that.relativeSize)
+                && (randomized == that.randomized)
+                && (interations == that.interations)
+                && (maxDepth == that.maxDepth)
+                && (defaultHorizontal == that.defaultHorizontal)
+                && (defaultVertical == that.defaultVertical);
     }
 
     @Override
     public String toString() {
+        // CHECKSTYLE:OOF
         return String.format("""
                 codeChartsConfiguration: {
-                        stringId: %s, initialSize: %s, timings: %s }""",
+                        stringId: %s, initialSize: %s, timings: %s, showGrid: %b, relativeSize: %b, randomized: %b, iterations %d, maxDepth: %s, defaultHorizontal: %s, defaultVertical: %s }""",
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 193).